django_dicom/reader/parser.py

```diff
@@ -337,7 +342,7 @@ def parse(self, element: DataElement):
             except KeyError:
                 return element.value
         except ValueError as e:
-            print(e.args)
+            # print(e.args)
```

Comment:
What is the purpose of this comment?

Reply:
I commented it out in order to see a clean test output, should be there.